### PR TITLE
Fix #6456: Perf getChildCount()

### LINKED ISSUE #6456: Performance: Use getChildCount() instead of size/empty
**Description**
Do not use `size()` or `isEmpty()` for any child counting using `getChildCount()` instead.

### PR DESCRIPTION
**Pull request by melloware**

### CODE REVIEW

File: src/main/java/org/primefaces/component/organigram/OrganigramHelper.java [hunk 1]
```diff
@@ -96,8 +96,8 @@ public static OrganigramNode findTreeNode(OrganigramNode searchRoot, OrganigramN
      * @param node The root node.
      */
     public static void buildRowKeys(OrganigramNode node) {
-        if (node.getChildren() != null && !node.getChildren().isEmpty()) {
-            for (int i = 0; i < node.getChildren().size(); i++) {
+        if (node.getChildren() != null && node.getChildCount() > 0) {
```
Comment (tandraschko) [inline src/main/java/org/primefaces/component/organigram/OrganigramHelper.java:99]: switch those statements

File: src/main/java/org/primefaces/component/organigram/OrganigramRenderer.java [hunk 2]
```diff
@@ -211,7 +211,7 @@ protected void renderNode(FacesContext context, ResponseWriter writer, Map<Strin
         requestMap.remove(organigram.getVar());
 
         // render child nodes
-        if (node.getChildren() != null && !node.getChildren().isEmpty()) {
+        if (node.getChildren() != null && node.getChildCount() > 0) {
```
Comment (tandraschko) [inline src/main/java/org/primefaces/component/organigram/OrganigramRenderer.java:214]: switch the statements

File: src/main/java/org/primefaces/component/organigram/OrganigramHelper.java (hunk 1 above)
Comment (melloware) [reply src/main/java/org/primefaces/component/organigram/OrganigramHelper.java:99]: actually do i need to check for NULL children at all of childCount>0 they can't be NULL right?  I think I can just remove the NULL check.

Comment (tandraschko) [reply src/main/java/org/primefaces/component/organigram/OrganigramHelper.java:99]: unsure, as its a model? :D i wouldnt remove it, just to be sure

**Comment by melloware** [comment]
I put it back the way it was because Chidlren can be NULL and the getChildCount of that model calls "children.size()" so it could throw an NPE.  For these two Organigram I just left the way it is to prevent NPE.

**Pull request merged**